apps/ui/src/stores/spaces.ts

```diff
@@ -34,10 +35,10 @@ export const useSpacesStore = defineStore('spaces', () => {
     };
   }
 
-  async function fetchSpaces(spaceIds: string[]) {
-    const spaces = await getSpaces({
-      id_in: spaceIds
-    });
+  async function fetchSpaces(input: string[] | SpacesFilter) {
+    const spaces = await getSpaces(
+      Array.isArray(input) ? { id_in: input } : input
```

Comment:
But we are not passing anything other than array any where? we need this condition?